cmd/mutagen/main.go

```diff
@@ -33,6 +34,7 @@ func rootMain(command *cobra.Command, _ []string) error {
 // rootCommand is the root command.
 var rootCommand = &cobra.Command{
 	Use:          "mutagen",
+	Version: mutagen.Version,
```

Comment:
Can you fix the alignment here so all values start at the same column?

Reply:
Done. Forgot I didn't have gofmt set up in a filewatcher for this project.